spec/formatting.md

```diff
@@ -590,3 +620,21 @@ rather than the _expression_ in the _selector_ or _pattern_.
 > ```
 > The value is {|horse|}.
 > ```
+
+When there are multiple _declarations_ for the same _variable_,
+the fallback string is formatted based on the _expression_ of
+the first declaration of that _variable_.
+
+> For example, attempting to format the following message:
+>
+> ```
+> let $var = {|cart|}
+> let $var = {|horse|}
```

Comment:
The second declaration (`let $var = {|horse|}`) would be an error.

Reply:
@mihnita I can't find the context for this comment since I've since pushed more commits -- I'm "resolving", but if the issue is still there, please comment again.